explainers/signed-exchange-subresource-subtitution-explainer.md

```diff
@@ -0,0 +1,405 @@
+personally-identifiable information or information derived thereof?
+    - Signed Exchange should not include personal information.
+    - Any personal information that was incorrectly included in a signed
+    exchange would be the information of the aggregator that fetched the SXG,
+    and not the end user.
+    - The use of `<link rel=alternate>` to identify the SXG for the current page
+    could inform the UA to omit credentials in fetching that SXG, which would
+    prevent any personal information from being accidentally included.
+1. How does this specification deal with sensitive information?
+    - Signed Exchange should not include sensitive information.
+    - The state of the cache for another origin is potentially sensitive, and
+    this specification avoids exposing it by making the decision to fetch an
+    alternative not depend on the presence or absence of the subresource in its
+    cache.
+1. Does this specification introduce new state for an origin that persists
+across browsing sessions?
+    - Prefetched resources, including signed exchanges, are stored to the HTTP
+    cache as normal, but the association of a signed exchange with its contained
+    resource is not persisted. Right now, the contained resource is not
+    independently stored in the HTTP cache, although that decision may be
+    revisited.
+1. What information from the underlying platform, e.g. configuration data, is
+exposed by this specification to an origin?
+    - The use of Variants exposes the UA's content negotiation preferences to
+    the aggregator's origin, but that's already exposed by the UA's Accept
+    headers.
+1. Does this specification allow an origin access to sensors on a user’s device
+    - No
+1. What data does this specification expose to an origin? Please also document
+what data is identical to data exposed by other features, in the same or
+different contexts.
+    - The existence of the alternative signed exchange in HTTP Cache is exposed.
+    - But this is the same as the existing behavior when directly prefetching the
+    signed exchange using `<link rel=prelfetch>`.
+1. Does this specification enable new script execution/loading mechanisms?
+    - This specification introduces a new script loading path, from prefetched
+    signed exchange. The existing security checks such as CSP/CORP must be
+    applied as if the script is loaded from the original URL.
+1. Does this specification allow an origin to access other devices?
+    - No
+1. Does this specification allow an origin some measure of control over a user
+agent’s native UI?
+    - No
+1. What temporary identifiers might this this specification create or expose to
+the web?
+    - None
+1. How does this specification distinguish between behavior in first-party and
+third-party contexts?
+    - This feature treats all entities (the distributor of signed exchange, the
+    publisher site, the origin of subresource URL) as third-party origins.
```

Comment:
I think the publisher's site winds up treated as a first-party origin, since it gets to be top-level after the navigation.

The behavior doesn't line neatly up under the heading of first- vs third-party, but I think we're trying to treat the distributor as a third-party, and of course cross-origin subresources are third-parties.

Reply:
Done.